pkg/iac/terraform/state/test/google_compute_router/terraform.tfstate

```diff
@@ -0,0 +1,173 @@
+{
+  "version": 4,
+  "terraform_version": "0.15.5",
+  "serial": 6,
+  "lineage": "5dbb3a54-27a7-9a8d-0c76-8d44f1873015",
+  "outputs": {},
+  "resources": [
+    {
+      "mode": "managed",
+      "type": "google_compute_network",
```

Comment:
You can remove this resource from the state, we want to keep this test minimalist even if we have an inconsistent state.